cpp/src/io/parquet/rle_stream.cuh

```diff
@@ -226,31 +247,12 @@ struct rle_stream {
     while (((decode_index == -1 && fill_index < num_rle_stream_decode_warps) ||
             fill_index < decode_index + run_buffer_size) &&
            cur < end) {
-      auto& run = runs[rolling_index<run_buffer_size>(fill_index)];
-
       // Encoding::RLE
+      auto& run           = runs[rolling_index<run_buffer_size>(fill_index)];
+      int const run_bytes = get_rle_run_info(run);
```

Comment:
It's somewhat non-cudf idiomatic to modify run in-place here.  It's fine, but maybe add a comment that it's happening.

Reply:
Done